baggage-processor/README.md

```diff
@@ -42,8 +42,18 @@ Pattern pattern = Pattern.compile("^key.+");
 new BaggageSpanProcessor(baggageKey -> pattern.matcher(baggageKey).matches())
 ```
 
+## Usage with SDK auto-configuration
+
+If you are using the OpenTelemetry SDK auto-configuration, you can add the span processor this
+library to configure the span processor.
+
+| Property                                  | Description                                                                                     |
+|-------------------------------------------|-------------------------------------------------------------------------------------------------|
+| otel.traces.baggage-to-attributes.include | Add baggage items to span attributes, e.g. `key1,key2` or `*` to add all baggage items as keys. |
```

Comment:
I'm unsure about `baggage-to-attributes` but also unsure if there's a convention for the property name so can't really offer a better name. Maybe `copy-baggage-attributes`?

Small suggestion for property description.
```suggestion
| otel.traces.baggage-to-attributes.include | Add baggage entries as span attributes, e.g. `key1,key2` or `*` to add all baggage items as keys. |
```

Reply:
I want to check what other languages have done

Reply:
I don't believe other languages have an automatic configuration component that can use the baggage span processor yet.